core/src/main/java/uk/gov/justice/services/core/enveloper/Enveloper.java

```diff
@@ -37,15 +40,15 @@
 @ApplicationScoped
 public class Enveloper {
 
-    private final ObjectToJsonValueConverter objectToJsonValueConverter;
+    Clock clock;
 
-    //for CDI proxying only
-    protected Enveloper() {
-        this.objectToJsonValueConverter = null;
-    }
+    ObjectToJsonValueConverter objectToJsonValueConverter;
+
+    public Enveloper() {}
```

Comment:
no default constructor results in: 

SEVERE - CDI Beans module deployment failed
org.apache.webbeans.exception.WebBeansDeploymentException: javax.enterprise.inject.UnproxyableResolutionException: WebBeans with api type with normal scope must be proxyable.
uk.gov.justice.services.core.enveloper.Enveloper has no explicit no-arg constructor!A public or protected constructor without args is required!
	at org.apache.webbeans.config.BeansDeployer.deploy(BeansDeployer.java:334)
	at org.apache.openejb.cdi.OpenEJBLifecycle.startApplication(OpenEJBLifecycle.java:196)
	at org.apache.openejb.cdi.ThreadSingletonServiceImpl.initialize(ThreadSingletonServiceImpl.java:193)
	at org.apache.openejb.cdi.CdiBuilder.build(CdiBuilder.java:41)
	at org.apache.openejb.assembler.classic.Assembler.createApplication(Assembler.java:895)
	at org.apache.openejb.assembler.classic.Assembler.createApplication(Assembler.java:694)
	at org.apache.openejb.assembler.classic.Assembler.createApplication(Assembler.java:690)
	at org.apache.openejb.testing.ApplicationComposers.deployApp(ApplicationComposers.java:721)
	at org.apache.openejb.testing.ApplicationComposers.before(ApplicationComposers.java:383)
	at org.apache.openejb.testing.ApplicationComposers.evaluate(ApplicationComposers.java:1060)
	at org.apache.openejb.junit.DeployApplication.evaluate(DeployApplication.java:40)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:367)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:274)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:161)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
Caused by: javax.enterprise.inject.UnproxyableResolutionException: WebBeans with api type with normal scope must be proxyable.
uk.gov.justice.services.core.enveloper.Enveloper has no explicit no-arg constructor!A public or protected constructor without args is required!
	at org.apache.webbeans.util.InjectionExceptionUtil.createUnproxyableResolutionException(InjectionExceptionUtil.java:46)
	at org.apache.webbeans.util.InjectionExceptionUtil.throwUnproxyableResolutionException(InjectionExceptionUtil.java:41)
	at org.apache.webbeans.config.DeploymentValidationService.validateProxyable(DeploymentValidationService.java:143)
	at org.apache.webbeans.component.creation.ManagedBeanBuilder.getBean(ManagedBeanBuilder.java:70)
	at org.apache.webbeans.config.BeansDeployer.defineManagedBean(BeansDeployer.java:1629)
	at org.apache.webbeans.config.BeansDeployer.deploySingleAnnotatedType(BeansDeployer.java:1255)
	at org.apache.webbeans.config.BeansDeployer.deployFromBeanAttributes(BeansDeployer.java:1211)
	at org.apache.webbeans.config.BeansDeployer.deploy(BeansDeployer.java:292)
	... 26 more